### DISCUSSION TITLE
Some error occured enter the one-time setup code

### DISCUSSION THREAD

**Discussion by shivarm** [profilecity/vidur#131] [Q&A] [answered]
I copy and paste the following key while running the project and got some error

![image](https://github.com/user-attachments/assets/69604fd8-69fd-458b-9de6-6e3b45bc57e9)

I Copy code from terminal you can verify..

![image](https://github.com/user-attachments/assets/b1c76189-2348-4d14-a6d0-b3c773b80f9b)

cc/ @amandesai01

**Comment by amandesai01** [comment, marked as answer]
yes, that is known issue. visit /login first

**Reply by shivarm** [reply]
@amandesai01 visited then redirected to `http://localhost:3001/onboarding` enter the key then still have same err

**Reply by amandesai01** [reply]
Open network tab and show error

**Reply by shivarm** [reply]
> yes, that is known issue. visit /login first

worked
